### PR TITLE
Updates to main.rst - Python 3.6 and Windows

### PR DESCRIPTION
**Pull request by mskadu**
I have made some further changes to make it clearer that Python 3.6 is recommended and that v2.7 is the oldest tested version. And as discussed in #2460 I have made some Windows specific revisions aimed at making this friendlier for fellow community members :)

Feedback welcome!

### CODE REVIEW

File: docs/guides/main.rst [hunk 1]
```diff
@@ -9,10 +9,9 @@ collection better.
 Installing
 ----------
 
-You will need Python.
-Beets works on `Python 2.7`_ and Python 3.4 or later.
+You will need `Python 3.6`_. But Beet also works with Python 2.7 and later.
```
Comment (sampsyo) [inline docs/guides/main.rst]: Python 3.4 actually works fine. 3.6 is just what we recommend, especially for Windows.

File: docs/guides/main.rst [hunk 2]
```diff
@@ -46,6 +45,8 @@ Beets works on `Python 2.7`_ and Python 3.4 or later.
 
 * On **NixOS**, run ``nix-env -i beets``.
 
+* On most versions of **Windows**, there is a `installer`_ available on the Python website.
```
Comment (sampsyo) [inline docs/guides/main.rst]: In this list, we're pointing toward places where you can install *beets* from the package manager, not just Python.

File: docs/guides/main.rst [hunk 3]
```diff
@@ -139,6 +146,11 @@ place to start::
     directory: ~/music
     library: ~/data/musiclibrary.db
 
+For Windows user, this would look like::
+
+    directory: "D:\\My Music Files"
+    library: "D:\\My Music Files\\Musiclibrary.db"
```
Comment (sampsyo) [inline docs/guides/main.rst]: The quotes and double-backslashes aren't actually necessary—they are harmless, but I think we can recommend that people don't need to do that, thanks to YAML's elegant syntax.

Comment (sampsyo) [inline docs/guides/main.rst]: Also, for what it's worth, you can still use the unixy-looking paths shown just above: `~/music` will work just fine and use a directory called "music" in your `%HOME%` directory.

Comment (mskadu) [inline docs/guides/main.rst]: IMHO, unixy paths are not natural to dominantly Windows users. And translating Windows paths to Unix format is even less so. When I read the guide, that was the first question that popped up in my mind. 

May I suggest we add an example that shows how paths with spaces in them are used? Or if they can be used as-is, worth explicitly clarifying that in the text?

Comment (sampsyo) [inline docs/guides/main.rst]: Right; you can use Windowsy paths too—I just wanted to point out that either works fine. And spaces work fine too; no need to handle them any differently.

**Pull request closed without merging**